x-pack/plugins/fleet/common/types/rest_spec/epm.ts

```diff
@@ -197,3 +199,14 @@ export interface DeletePackageResponse {
 export interface GetVerificationKeyIdResponse {
   id: string | null;
 }
+
+export interface GetBulkAssetsRequest {
+  body: {
+    objects: AssetSOObject[];
```

Comment:
nit: we could rename `objects` to `assets`

Reply:
I thought about it, but isn't confusing? We are fetching assets, I'm not sure how to call the objects {id:.., type:...} used to get them

Reply:
Sounds good!